bokeh/models/tools.py

```diff
@@ -371,8 +371,22 @@ class HoverTool(Tool):
     off by setting ``tooltips=None``.
 
     .. warning::
-        Point hit testing is not currently available on all glyphs. Hover tool
-        currently does not work with line or image type glyphs.
+        Hover tool does not currently work with the following glyphs:
+            * annulus
+            * arc
+            * bezier
+            * gear
+            * image
+            * image_rgba
+            * image_url
+            * multi_line
+            * oval
+            * patch
+            * quad
```

Comment:
Doesn't work for me (you can run the updated glyphs example in this PR to see). However, having had a look it seems to break for a different reason. The others all have a console message "Point testing...." this one says: `Error: Permission denied to access property "type"` - maybe that's a regression.